djangocms_versioning/admin.py

```diff
@@ -53,27 +53,17 @@ def get_queryset(self, request):
         for specifying filters that work without being shown in the UI
         along with filter choices.
         """
+        qs = super().get_queryset(request)
         try:
             grouper = int(request.GET.get(GROUPER_PARAM))
         except (TypeError, ValueError):
-            raise IncorrectLookupParameters("Invalid grouper")
-        content_type_id = request.GET.get('content_type_id')
-        try:
-            content_type = ContentType.objects.get(pk=content_type_id)
-            model = content_type.model_class()
-        except (ContentType.DoesNotExist, ValueError):
-            raise IncorrectLookupParameters("Invalid content_type_id")
+            return qs
```

Comment:
Correct me if I'm wrong, but I don't think you have a test for this. I.e. you check that in this case a grouper form is displayed, but you never check that a full queryset is returned.